Assets/Scripts/Copilots/Copilots.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Copilots : MonoBehaviour
+{
+    public static Copilots singleton;
```

Comment:
Noticed you made this a singleton. Can you add a class comment as to why because rn i'm confused as to, well, why

Reply:
Mostly cuz I didn't want to have a reference to it on this script as well as I figured there shouldn't ever be more than one of hem anyways since its going to be attached to the player